src/logion-chain/Vault.ts

```diff
@@ -17,41 +18,69 @@ export function getVaultAddress(requesterAddress: string, recoveryConfig: Recove
     return vaultAddress;
 }
 
-export interface RequestVaultTransferParameters extends ExtrinsicSubmissionParameters {
+export interface BuildRequestVaultTransferParameters {
     api: ApiPromise;
     recoveryConfig: RecoveryConfig;
     amount: PrefixedNumber;
     destination: string;
+    errorCallback: ErrorCallback;
```

Comment:
Not sure to understand why the callback would need to be passed here. But I would add the requesterAddress (instead of creating a type union later).

Reply:
> Not sure to understand why the callback would need to be passed here

errorCallback Removed.

> But I would add the requesterAddress (instead of creating a type union later).

That was done on purpose - I wanted the parameter `requesterAddress` to be available only for recovery.

Reply:
OK